### ISSUE TITLE
Citations for multiple paragraphs

### ISSUE THREAD

**Issue by cangyuyao** [mdickin/vscode-markdown-shortcuts#43]
When selecting multiple paragraphs, the current citation works neglecting the blank lines in-between:  
```
> first paragraph

> second paragraph
```

Rendered: 
> first paragraph

> second paragraph

But most time I want to cite them in the same blockquote like: 

```
> first paragraph
> 
> second paragraph
```

> first paragraph
> 
> second paragraph

**Comment by mdickin** [comment]
Good point, that's not a scenario I tested (clearly). Seems like that would be the expected behavior most of the time

**Comment by mdickin** [comment]
This has been resolved in v0.10.0. Thanks for the feedback!
